sentry-android-core/src/main/java/io/sentry/android/core/AppComponentsBreadcrumbsIntegration.java

```diff
@@ -53,6 +54,10 @@ public void register(final @NotNull IHub hub, final @NotNull SentryOptions optio
         options
             .getLogger()
             .log(SentryLevel.DEBUG, "AppComponentsBreadcrumbsIntegration installed.");
+        final SdkVersion sdkVersion = this.options.getSdkVersion();
+        if (sdkVersion != null) {
+          sdkVersion.addIntegration("AppComponents");
```

Comment:
`m`: Maybe we should keep the `Breadcrumbs` suffix for all integration, so we know it's about breadcrumbs?
```suggestion
          sdkVersion.addIntegration("AppComponentsBreadcrumbs");
```